truss/cli/cli.py

```diff
@@ -1182,7 +1179,7 @@ def push(
         tr,
         model_name=model_name,
         publish=publish,
-        trusted=trusted,
+        trusted=True,
```

Comment:
I think it would be cleaner to strip this parameter/argument out of any functions / APIs and mace a change on the backend that hard-codes it to true.